plugins/tasks/http/src/main/java/com/walmartlabs/concord/plugins/http/SimpleHttpClient.java

```diff
@@ -332,6 +333,7 @@ private static CloseableHttpClient createClient(Configuration cfg) throws Except
         return HttpClientBuilder.create()
                 .setConnectionManager(buildConnectionManager())
                 .setDefaultRequestConfig(c.build())
+                .setRedirectStrategy(new LaxRedirectStrategy())
```

Comment:
This should be enabled based on the `followRedirects` option. Something like:
```java
RedirectStrategy redirectStrategy;
if (cfg.isFollowRedirects()) {
    redirectStrategy = new LaxRedirectStrategy();
} else {
    redirectStrategy = DefaultRedirectStrategy.INSTANCE;
}

return HttpClientBuilder.create()
        .setConnectionManager(buildConnectionManager())
        .setDefaultRequestConfig(c.build())
        .setRedirectStrategy(redirectStrategy)
        .build();
```

Reply:
@benbroadaway I have tested this. it works only when followRedirects is true, and doesn't when false. you can check the test cases too

Reply:
Aaah so it's already tied to `redirectsEnabled` in the apache `RequestConfig`. Got it.